tests/console/lvm_thin_check.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple LVM thin provisioning check
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use y2logsstep;
+use serial_terminal 'select_virtio_console';
+
+sub run {
+    select_virtio_console();
+    record_info('INFO', 'Print lvm setup');
+    assert_script_run 'lsblk';
+    assert_script_run 'lvmdiskscan';
+    assert_script_run 'lvscan';
+    assert_script_run 'lvs -a  | tee original_usage';
+    assert_script_run 'pvs -a';
+    # thin volume does not exceed thin pool size in our tests
+    my @volumes = split(/\n/, script_output q[lvscan | awk '{print $2}'| sed s/\'//g]);
+
+    foreach (@volumes) {
+        chomp;
+        assert_script_run "lvdisplay $_";
```

Comment:
@rwx788, please wait with the merge. I am still working on extra check over here.